plugins/lime-plugin-pirania/src/piraniaApi.js

```diff
@@ -0,0 +1,64 @@
+import api from "utils/uhttpd.service";
+
+export function listVouchers() {
+	const now = new Date().getTime() / 1000;
+	return api
+		.call("pirania", "list_vouchers", {})
+		.then((response) =>
+			response.vouchers.map((voucher) => {
+				const { is_active, expiration_date, duration_m } = voucher;
+				let status = is_active ? "used" : "available";
+
+				if ((expiration_date && expiration_date < now) || duration_m === 0)
+					status = "disabled";
+				return {
+					...voucher,
+					status
+				};
+			})
+		)
```

Comment:
This is how I'm checking the status of the voucher. Is this the best way? Related to: https://github.com/libremesh/lime-packages/pull/893#discussion_r738762672

Reply:
In https://github.com/libremesh/lime-packages/pull/893/commits/36784d36b468577cdc96f0052cad0b429c04a72f I've added the `status` attribute that may be one of 'available', 'active' or 'expired'